examples/bilingual_book.html

```diff
@@ -0,0 +1,254 @@
+<table border="1" class="dataframe">
```

Comment:
Do we really need to commit this file?
Github doesn't display it as a proper html anyway.